From cce4a256d7fb13f0f75591c2e90065d4c0aa181c Mon Sep 17 00:00:00 2001
From: James Forcier <csssuf@csssuf.net>
Date: Thu, 26 Apr 2018 22:07:11 -0700
Subject: [PATCH 2/6] window: don't handle pixel scaling

glfwGetWindowContentScale is in the still-unreleased glfw 3.3, so we
can't build with it.
---
 src/window.cpp | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src/window.cpp b/src/window.cpp
index bb72888..acbe157 100644
--- a/src/window.cpp
+++ b/src/window.cpp
@@ -446,16 +446,6 @@ void windowRun() {
 			lastWindowTitle = windowTitle;
 		}
 
-		// Get desired scaling
-		float pixelRatio;
-		glfwGetWindowContentScale(gWindow, &pixelRatio, NULL);
-		pixelRatio = roundf(pixelRatio);
-		if (pixelRatio != gPixelRatio) {
-			EventZoom eZoom;
-			gScene->onZoom(eZoom);
-			gPixelRatio = pixelRatio;
-		}
-
 		// Get framebuffer/window ratio
 		int width, height;
 		glfwGetFramebufferSize(gWindow, &width, &height);
-- 
2.17.0

